utils/api/configurations.js

```diff
@@ -69,7 +69,7 @@ export const configureRequests = ({ data, path }) => {
 export const configureErrors = (errors) => {
   const env = process.env.NODE_ENV
   const remainingErrors = errors
-    .filter(error => Object.keys(error).length)
+    .filter(error => error && Object.keys(error).length)
```

Comment:
```suggestion
    .filter(error => Object.keys(error).length)
```

Reply:
we don't have to explicitly check for the truthiness of an error here because `Object.keys` is handling it.

```
[{}, false, { a: 'a' }].filter(error => Object.keys(error).length)
=> Array [ { a: "a" } ]
```

Reply:
but what if the array ends up with an item that is undefined? like
[{}, false, { a: 'a' }, undefined]

That is what I was running into in the test that prompted me to change this. idk if its possible for this array of errors to contain one that is undefined, but it was happening in the test. so I figured adding a check there to make sure the error was not undefined was basically some TDD.

Reply:
when it was undefined, Object.keys(error) was v angry